src/consts.ts

```diff
@@ -9,27 +9,27 @@ export const EN_UI_LABELS = {
   SystemDefault: "Current state: System default",
   Indefinitely: "Current state: Indefinitely",
   SimulatorActive: "Stimulator is active",
-  ConfirmClose: "Confirm close?",
-  ConfirmCloseBody: "Stimulator is active, do you want to close?",
+  ConfirmClose: "Close Stimulator?",
+  ConfirmCloseBody: "Stimulator is active, do you want to close it?",
   Cancel: "Cancel",
   Close: "Close",
   Preferences: "Preferences",
   Theme: "Theme",
-  ThemeSystem: "System theme",
+  ThemeSystem: "System Theme",
   ThemeLight: "Light",
   ThemeDark: "Dark",
-  EnableExistConfirmation: "Enable closing confirmation",
+  EnableExistConfirmation: "Closing Confirmation",
   EnableExistConfirmationSubTitle:
-    "Confirmation asked when automatic suspending disabling is active",
+    "Ask for confirmation to close when Stimulator is active",
   KeyboardShortcuts: "Keyboard Shortcuts",
   General: "General",
-  MainMenu: "Open menu",
+  MainMenu: "Open Menu",
   Quit: "Quit",
   About: "About Stimulator",
   UnsupportedSystem: "Unsupported System",
   UnsupportedSystemBody:
-    "Your desktop enviorement doesn't support `org.freedesktop.portal.Inhibit` portal, click close to quit",
-  EnableTrayIcon: "Enable tray icon and backgrounding on exit",
+    "Your desktop environment doesn't support Stimulator, click close to quit",
+  EnableTrayIcon: "Run in Background",
```

Comment:
This one doesn't mention tray icon at all now, is that intended?

Reply:
It is, but I'm not sure if it complies with the technical implementation, does Stimulator use the background portal?